src/layouts/shortcodes/Button.tsx

```diff
@@ -1,20 +1,23 @@
 import React from "react";
+import type Tab from "./Tab";
```

Comment:
I think this may have been added by mistake? I don’t see it used anywhere in the code below.